MovieApp/MoviePictureCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  MovieCatergoyCell.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 02.05.2023..
+//
+
+import Foundation
+import UIKit
+import PureLayout
+import MovieAppData
+
+class MoviePictureCell: UICollectionViewCell {
```

Comment:
Also check [prepareForReuse](https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse) it is a Cell method that should be used to invalidate all of the set data, so once the cell is reused, new data has an empty canvas

This is visible if you scroll the list fast, you will notice that some of the cells display an old image and once the new one is downloaded it is replaced